eth/consensus/pow.py

```diff
@@ -62,8 +73,19 @@ def check_pow(block_number: int,
     mining_output = hashimoto_light(
         block_number, cache, mining_hash, big_endian_to_int(nonce))
     if mining_output[b'mix digest'] != mix_hash:
-        raise ValidationError("mix hash mismatch; {0} != {1}".format(
-            encode_hex(mining_output[b'mix digest']), encode_hex(mix_hash)))
+        raise ValidationError(
+            "mix hash mismatch; expected: {} != actual: {}. "
```

Comment:
f-string?

Reply:
py-evm is still supporting py3.5 unfortunately, afaik

Reply:
I didn't see what codebase I was in :disappointed: